src/components/DappBrowser/Homepage.tsx

```diff
@@ -95,13 +97,15 @@ const Trending = ({ goToUrl }: { goToUrl: (url: string) => void }) => {
           decelerationRate="fast"
           disableIntervalMomentum
           showsHorizontalScrollIndicator={false}
-          snapToOffsets={dapps.map((_, index) => index * (CARD_WIDTH + CARD_PADDING))}
+          snapToOffsets={data.dApps.map((_, index) => index * (CARD_WIDTH + CARD_PADDING))}
         >
           <Inset space="24px">
             <Box flexDirection="row" gap={CARD_PADDING}>
-              {dapps.map((site, index) => (
-                <Card goToUrl={goToUrl} index={index} key={site.url} site={{ ...site, image: site.iconUrl }} />
-              ))}
+              {data.dApps
+                .filter((dApp): dApp is DApp => dApp !== null)
```

Comment:
Shouldn't this be handled in `useTrendingDApps`?

Reply:
typescript tings.. but essentially yes